StartScreen/LuisJavascript.js

```diff
@@ -0,0 +1,164 @@
+console.log("Windows 10 BYOL agree button was clicked"); })
+document.getElementById("acceptButton").click();
+*/
+}
+checkVMclicks();
+
+
+function clickVM(){
+
+function region(){
+document.getElementsByClassName("regionSelect")[0].click();
+document.getElementsByClassName("regionSelect")[1].click();
+document.getElementsByClassName("regionSelect")[2].click();
+document.getElementsByClassName("regionSelect")[1].click();
+}
+region();
+
+function OSoptions(){
+document.getElementsByClassName("strong tab")[3,2,1,0].click();
+return true;
+};
+if ( OSoptions() === true ) {
+var elements = document.getElementsByClassName("strong tab");
+
+for (i = 0; i < elements.length; i++) {
+    console.log(elements[i].innerHTML + " " + "was clicked");
+}
+};
+
+document.getElementsByClassName("strong tab")[0].click();
+document.getElementsByClassName("os_block")[0,1].click();
+document.getElementsByClassName("os_block")[2].click();
+
+var agree = document.querySelectorAll(' #acceptButton ')
+
+agree[0].addEventListener('click', function(event) { console.log(event.currentTarget);
+console.log("Windows 10 BYOL agree button was clicked"); })
+
+document.getElementById("acceptButton").click();
+
+
+document.getElementsByClassName("strong tab")[1].click();
+document.getElementsByClassName("templateTileIcon")[0,1].click();
+
+document.getElementsByClassName("strong tab")[2].click();
+document.getElementsByClassName("templateTileIcon")[0,1,2,3].click();
+
+document.getElementsByClassName("strong tab")[3].click();
+document.getElementsByClassName("strong tab")[0].click();
+document.getElementsByClassName("os_block")[0].click();
+
+
+function switchRight(){
+document.getElementsByClassName("custom-switch")[1].click();
+document.getElementsByClassName("vmType")[0,1,2,3].click();
+}
+switchRight();
+
+function tilesForRightSwitch(){
+document.getElementsByClassName("vmType")[0,1,2,3,4,5,6,7].click();
+}
+tilesForRightSwitch();
+
+
+function switchLeft(){
+document.getElementsByClassName("custom-switch")[1].click();
+document.getElementsByClassName("vmType")[0,1,2,3].click();
+}
+
+switchLeft();
+
+function tilesForLeftSwitch(){
+document.getElementsByClassName("vmType")[0,1,2,3,4,5,6,7].click();
+}
+tilesForLeftSwitch();
+
+function checkAllwithDefault(){
+document.getElementsByClassName("circleOpt")[2].click();
+}
+checkAllwithDefault();
+
+function uncheckAll(){
+document.getElementsByClassName("circleOpt")[2].click();
+document.getElementsByClassName("circleOpt")[1].click();
+document.getElementsByClassName("circleOpt")[0].click();
+}
+uncheckAll()
+
+function checkDefault(){
+document.getElementsByClassName("circleOpt")[1].click();
+document.getElementsByClassName("circleOpt")[0].click();
+}
+checkDefault();
+
+function createPaperspace(){
+document.getElementById("submitPaymentInfo").click();
+}
+createPaperspace();
+
+}
+clickVM();
+
+if(
+window.location.href === 'https://www.paperspace.com/console/machines'
```

Comment:
sometimes what it'll do is change the URL and say "machines/'.

Reply:
Thank you for reviewing my code, hope there wasn't any confusions in my code and will keep in mind about adding that advice the next time I edit this code.